tests/VolleyManagement.UnitTests/Services/GameService/Fixture/GameServiceTestFixture.cs

```diff
@@ -331,8 +333,8 @@ public GameServiceTestFixture TestGamesWithResultInTwoWeeksTwoDivisionsTwoGames(
                         new Score(25, 0),
                         new Score(25, 0),
                         new Score(25, 0),
-                        new Score(0, 0),
-                        new Score(0, 0),
+                        new Score(0,  0),
```

Comment:
Why?

Reply:
The code in the file formatted incorrectly. Please format.

Reply:
It is formatted into table like layout. So same arguments are aligned across lines. Do you think I should avoid that?

Reply:
I prefer standard formatting (which we get while use 'CTRL+K CTRL+D' shortcut). It is more familiar to me.